x/harvest/keeper/liquidation.go

```diff
@@ -16,23 +18,29 @@ type LiqData struct {
 
 // AttemptIndexLiquidations attempts to liquidate the lowest LTV borrows
 func (k Keeper) AttemptIndexLiquidations(ctx sdk.Context) error {
-	// use moneyMarketCache := map[string]types.MoneyMarket{}
-
-	// Iterate over index
-	//		Get borrower's address
-	//		Use borrower's address to fetch borrow object
-	//		Calculate outstanding interest and add to borrow balances
-	//		Use current asset prices from pricefeed to calculate current LTV for each asset
-	//		If LTV of any asset is over the max, liquidate it by
-	//			Sending coins to auction module
-	//			(?) Removing borrow from the store
-	//			(?) Removing borrow LTV from LTV index
+	params := k.GetParams(ctx)
+	borrowers := k.GetLtvIndexSlice(ctx, params.CheckLtvIndexCount)
+
+	// Sort items in slice to ensure deterministic auction order
+	sortedBorrowers := addressSort(borrowers)
```

Comment:
I don't think sorting is necessary for deterministic order

Reply:
Removed in https://github.com/Kava-Labs/kava/pull/743/commits/7f90920acafd1cdc5178df19a9a07d08bbda6a66, https://github.com/Kava-Labs/kava/pull/743/commits/d110c36bbeb34abd432abae77e8b446efbe29f85